### PR TITLE
Chore: Main README update

### PR DESCRIPTION
**Pull request by rihardsgravis**
Update the main repository README.md content

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,126 +1,201 @@
...
-## What is Robo?
+# On Linux or Windows
+pip install robocorp-action-server
+```
+
+Bootstrap a new project from a template. You’ll be prompted for a name of the project:
 
-Robo is both a CLI for managing your Python projects, and a set of libraries for automating your tasks. It handles your entire automation development lifecycle. Robo's key features are:
+```sh
+action-server new
```
Comment (bendersej) [inline README.md:39]:
```suggestion
action-server new --name "my-project"
```

[Re: this discussion](https://robocorptechnologies.slack.com/archives/C04SAQERGCF/p1704469960754919)

This way it nicely aligns it with the next command: `cd my-project` and as a first time user, it's clear

File: README.md [hunk 2]
```diff
@@ -1,126 +1,201 @@
...
 
-- **Isolated environments.** Get started without installing any other tools, not even Python. Robo automatically creates fully isolated environments for projects, which can be reproduced anywhere else.
-- **Batteries included.** Robo ships with a broad set of libraries for automating common tasks, be it browser automation, document processing, or navigating desktop applications.
-- **Reduce boilerplate.** Turn your Python snippet into a runnable automation with minimal effort. Robo can be used to execute your code in other environments, and deployed in Control Room without any infrastructure work.
-- **Troubleshoot easily.** Robo automatically traces your automation task, and create an easy-to-read report of what happened. You can quickly see if something went wrong - and how to fix it.
+Navigate to the freshly created project folder and start the server:
 
-Robo is designed to function as a single tool that helps you more easily get things done. It is entirely compatible with the PyPI and Conda ecosystems, but also brings in a set of libraries designed from the ground up to be focused on automation.
+```sh
+cd my-project
```
Comment (bendersej) [inline README.md]:
```suggestion
cd my-first-action-server
```

WDYT? To cement the term even more and gently hint at the fact that it's your first, but many will follow!

File: README.md [hunk 3]
```diff
@@ -1,126 +1,201 @@
...
-- [Licence](#licence)
+Robocorp is the easiest way to extend the capabilities of AI agents, assistants and copilots with custom actions, written in Python. Create and deploy tools, skills, loaders and plugins that securely connects any AI Assistant platform to your data and applications.
 
-</details>
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="./docs/include/robocorp-flow-dark.webp">
+  <img alt="Text changing depending on mode. Light: 'So light!' Dark: 'So dark!'" src="./docs/include/robocorp-flow-light.webp">
+</picture>
+
+Looking for a replacement to RPA? Head over to our [Enterprise Python Automation site](https://robocorp.com/docs/quickstart-guide) for more.
```
Comment (bendersej) [inline README.md:20]: FYI, @tonnitommi 

[Related to your comment on Notion](https://www.notion.so/robocorptechnologies/New-Robocorp-repo-07d02cb760a9450f8ce83e6f5e5304ab):
> Internal battle in my head where to link. I am thinking Getting started on Docs. ANy better ideas?

File: README.md (hunk 1 above)
Comment (rihardsgravis) [reply README.md:39]: First time users should not learn the flags though and trust the interactive console 🤔  If the example is shown as this, it looks like this is a mandatory flag (it's not).

Comment (bendersej) [reply README.md:39]: I'd argue that optimising for _time to running locally_ is another valid approach: if your readme tells me that I have only one command to run, but then you ask me about (completely made up examples):
- Do you want a linter?
- Do you want a formatter?
- Which formatter do you want?

I'll be overwhelmed – I was just planning on "getting started".

As opposed to a sane default / one-liner that doesn't change over time (as the CLI may have new options), which provides with a consistent 0 to dev experience.


_To be clear: providing another PoV, completely fine with the current command_

Comment (rihardsgravis) [reply README.md:39]: Right, but assuming that this is a command you would use more than once in a life time – when using such command would you:
- Try to remember the one liner
- Come back to the repo to find the one liner
- Use the interactive console

Meaning – what is our goal – get the user started as fast a possible once or teach the user how to get started fast repeatedly.

Comment (bendersej) [reply README.md:39]: Fair enough! Sold!

File: README.md [hunk 4]
```diff
@@ -1,126 +1,201 @@
...
-The above minimum example defines a task, which can then be executed with `robo run`. Once execution is finished, you can view the log generated (by default) to log.html file.
+---
+
+## Add Action Server as a Toolkit to [🦜️🔗 LangChain](https://github.com/robocorp/langchain)
+
+Robocorp Action Server comes with everything needed to connect it to your Langchain AI app project. The easiest way is to start with the template, provided in the Langchain project. Here’s how to do it:
+
+```sh
+# Install LangChain cli tool if not already there
+pip install langchain-cli
```
Comment (bendersej) [inline README.md:105]: Should we offer the `brew` alternative here as well? Or that's on purpose

Comment (rihardsgravis) [reply README.md:105]: There is no brew `langchain-cli` package I think, or you meant something else?

File: README.md (hunk 1 above)
Comment (kariharju) [reply README.md:39]: As we might also still change the cli args and flags I'd consider this good in current from, resolving this

File: README.md (hunk 3 above)
Comment (kariharju) [reply README.md:20]: It is either this or `https://robocorp.com/docs`, at this time I'd just go with the one here

File: README.md [hunk 5]
```diff
@@ -1,126 +1,195 @@
...
 
-> **Warning**<br/>
-> Robo CLI is not yet recommended for production use, while libraries are ready for prime time. Refer to Robocorp's production tooling for managing Python projects [here](https://robocorp.com/docs/quickstart-guide/python).
+Robocorp stack is hands down the easiest way to give AI agents more capabilities. It’s an end-to-end stack supporting every type of connection between AI and your apps and data. You are in control where to run the code and everything is built for easiness, security, and scalability.
 
-The `robo` CLI is used to manage and run your projects during development, and it bundles your project for deployment.
+- 🔐 **Decouple AI and Actions that touches your data/apps** - Clarity and security with segregation of duties between your AI agent and code that touches your data and apps. Build `@action` and use from multiple AI frameworks.
+- 🏎️ **Develop Actions faster with `robocorp` automation libraries** - Robocorp libraries and the Python ecosystem let's you act on anything - from data to API to Browser to Desktops.
```
Comment (sahokas) [inline README.md]: let's or lets ?

File: README.md [hunk 6]
```diff
@@ -1,126 +1,195 @@
...
+- 🕵️ **Observability out of the box** - Log and trace every `@action` run automatically without a single `print` statement.
```
Comment (sahokas) [inline README.md]: LangSmith

File: README.md [hunk 7]
```diff
@@ -1,126 +1,195 @@
...
-#### `robo new`
+## Inspo
 
-Create an entirely new Python project, isolated from the rest of your system. No need to install Python separately or manually manage virtual environments.
+Check out these example projects for inspiration.
 
-#### `robo run`
+- 🤡 Get a random joke or jokes per theme. Showcases how easy it is to work with APIs. (Coming soon, credit XX for the idea)
+- 🕸️ Open a local Playwright browser and make some Google searches. (Coming soon)
+- 🖥️ Fetch contents of `.txt` and `.pdf` files from your local machine's folder in real time. (Coming soon)
```
Comment (sahokas) [inline README.md]: formulate a little bit differently to promote secure, strictly controlled access to the local machine resources (explicitly configured folder etc) ?

File: README.md [hunk 8]
```diff
@@ -1,126 +1,195 @@
...
 
-Run your defined Python tasks. Robo instruments the running code, and generates a pretty report of what was executed.
+Build more `@actions` and be awesome! We'd love to hear and see what have you built. Join our [Slack community](https://robocorp-developers.slack.com/) to share your work, or drop us a line at [tommi@robocorp.com](mailto:tommi@robocorp.com). We'll soon start showcasing the best of the community here!
```
Comment (sahokas) [inline README.md:174]: name a channel on slack?

File: README.md [hunk 9]
```diff
@@ -1,126 +1,195 @@
...
+# Install LangChain cli tool if not already there
+pip install langchain-cli
 
-## Libraries
+# Use the Action Server template
+langchain app new my-awesome-app --package action_server
 
-The framework includes Python libraries for configuring project entrypoints and controlling features such as a logging. The project also provides an extensive selection of libraries for the most common automation tasks, and anything else can be easily done through what is already available in the Python ecosystem.
+# Start the template locally
+langchain template serve
```
Comment (sahokas) [inline README.md]: note to self: fails on Sampo's Macbook -- to be investigated

Comment (rihardsgravis) [reply README.md]: The template is not published yet!

Comment (rihardsgravis) [reply README.md]: Published now and updated the steps

**Pull request merged**